rhombus/rhombus/scribblings/reference/filesystem.scrbl

```diff
@@ -72,3 +72,15 @@
  directory cannot be found.
 
 }
+
+@doc(
+  fun filesystem.list_directory(path :: PathString = Path.current_directory(),
+                                ~build_path: build = #false) :: List.of(Path)
```

Comment:
Would the keyword name `absolute` be better? The return signature is `List.of(Path)`, so it's building `Path`s whether `build_path` is true or false...

Reply:
It doesn't build an absolute path though.  It extends the `path` with the entry from the directory.

Reply:
`extend_path` may be better since the library doesn't use `extend` instead of `build`.

Reply:
I agree that `extend_path` sounds better.

The reason I proposed `absolute` is because the text says:

```
  If @rhombus(build) is @rhombus(#false) the resulting
  paths are all @rhombus(Path.Element)s; otherwise, the results are combined
  with @rhombus(path) to create @rhombus(Path.Absolute)s.
```

Is that not accurate?

Reply:
You're right I miswrote that.  Will fix.